docs/topic/access-creds/secrets.md

```diff
@@ -15,7 +15,7 @@ For example, see the [2i2c cluster directory](https://github.com/2i2c-org/infras
 All our secrets are encrypted with [`sops`](https://github.com/mozilla/sops).
 
 :::{seealso}
-For information about how to set up `sops`, see [the team compass documentation](tc:secrets:sops)
+For information about how to set up `sops`, see [the team compass documentation](https://compass.2i2c.org/engineering/secrets/)
```

Comment:
I presume (to be tested) something like this would have worked out (to avoid hardcoding):
```
[the team compass documentation](tc:engineering:secrets)
```
or more specifically (again, untested):
```
[the team compass documentation](tc:engineering:secrets:#sops-overview)
```
@choldgraf maybe know if this is actually valid 😉